setup.md

```diff
@@ -53,27 +51,35 @@ The benefit of using a compute cluster with an ESGF node is that the
 is locally stored on disk and accessible directly by the tool.
 Similarly, observational data would also be available at these sites.
 
-Here are a few options
-for compute clusters with ESGF nodes:
+Here are a few options for compute clusters with ESGF nodes:
 
-- CEDA-Jasmin (UK):
-- DKRZ (Germany):
+- [CEDA-Jasmin (UK)](#CEDA-Jasmin)
+- [DKRZ (Germany)](#DKRZ)
 - ETHZ (Switzerland):
 
 A full list of all ESGF nodes is available [here](https://esgf.llnl.gov/nodes.html).
 
-
 If you're running on a computing cluster without an ESGF node, such as 
 your local machine, or your institue machine, you will most likely 
 have to make a local copy of the data that you need.
 
-If neccesairy, data can be downloaded using the 
+While some data will be provided by your tutors, there is simply too much data 
+for your tutors to make it all available directly.
+
+If you do not have access to an ESGF node, data can be downloaded using the 
 [synda tool](https://prodiguer.github.io/synda/index.html).
- 
+
+Depending on which machine you plan to use, you will need to work through one set of the following instructions:
+- [CEDA-Jasmin (UK)](#CEDA-Jasmin)
```

Comment:
These links still don't work!

Reply:
Don't use capitals: `#ceda-jasmin` works for me, just like `#dkrz` and `#your-local-machine`.

Reply:
fixed